.github/workflows/scripts/compare_translations.py

```
@@ -0,0 +1,236 @@
+        with open(filepath, "r", encoding="utf-8") as file:
+            content = file.read()
+            if not content.strip():
+                raise ValueError(f"File {filepath} is empty.")
+            translation = json.loads(content)
+            flattened_translation = flatten_json(translation)
+        return flattened_translation
+    except json.JSONDecodeError as e:
+        raise ValueError(f"Error decoding JSON from file {filepath}: {e}")
+
+
+def check_translations(directory):
```

Ruff (0.12.2)

Missing return type annotation for public function `check_translations`

Add return type annotation: `None`

(ANN201)